app/taskmanager.py

```diff
@@ -1,11 +1,13 @@
 cont = "y"
 lst = list()
 while cont == "y":
+    item = input("\nEnter the list item: ")
+    lst.append(item)
+    cont = input("\nDo you want to continue?[y/n] :")
 
-    lst = lst.append(input("enter the list item: "))
```

Comment:
lst.append then assigning back to the lst caused issue. when i removed that it worked.